tests/ubpf_vm.rs

```diff
@@ -9,7 +9,6 @@
 // <https://github.com/iovisor/ubpf/tree/master/tests> (hyphen had to be replaced with underscores
 // as Rust will not accept them in function names). It is strongly advised to refer to the uBPF
 // version to understand what these program do.
-//
```

Comment:
Please keep the line :)

Reply:
Yes sorry about that, must have deleted it accidentally and didn't spot it in the diff. It was inserted back.